yarn-error.log

```diff
@@ -0,0 +1,2941 @@
+Arguments: 
```

Comment:
Please remove this whole file. put it in the gitignore and also run `git rm ....`